src/Triangle.ts

```diff
@@ -1,4 +1,42 @@
 import { Shape } from './Shape';
 import { Point } from "./Point";
 
-export class Triangle extends Shape {}
+export class Triangle extends Shape {
+    protected color: string;
+    protected filled: boolean;
```

Comment:
@dianalemen, why did you duplicate these fields from the parent class?

Reply:
just forgot to delete